manifests/services.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: minecraft-minecraft
+  labels:
+    app: minecraft-minecraft
+    chart: "minecraft-3.1.3"
+    release: "minecraft"
+    heritage: "Helm"
+  annotations:
+    {}
+spec:
+  clusterIP: 172.30.48.134
+  clusterIPs:
+  - 172.30.48.134
+  externalTrafficPolicy: Cluster
+  ports:
+  - nodePort: 31904
+    port: 25565
+    protocol: TCP
+    targetPort: 25565
+  selector:
+    app: minecraft-minecraft
+  sessionAffinity: None
+  type: NodePort
```

Comment:
I'd set the type to `LoadBalancer` unless you really need a `NodePort`. With `LoadBalacer` you don't need the `clusterIP`, `externalTrafficPolicy`, `sessionAffinity` and `nodePort` in `ports` properties.

With `LoadBalancer` you may need a `Route` resource to expose the service to external traffic - assigning a DNS entry for you.

Reply:
But with `LoadBalancer` it needs to HTTP traffic, right?

Reply:
Nope, `LoadBalancer` can be also raw TCP and UDP.